common/xml/convert_and_merge_composable_tests/pack_pattern-merge-type-into-name.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<xml>
+ <pb_type>
+  <direct>
+   <pack_pattern name="B" type="carry" />
```

Comment:
Might be worth adding a comment in the XSLT saying "When this VPR issue is resolved, can remove this transform."

Reply:
@litghost Correct - the idea is that VPR will be explicit about the pack_pattern types to prevent the current horrible confusion around carry / chain verse molecule patterns.

Reply:
Renamed to chain and molecule.